apps/network/tests/database/presets/user.py

```diff
@@ -0,0 +1,26 @@
+user_metrics = [
+    (
+        "awesome@mail.com",
+        "D2BC2F8D09990EBE87C809684FD78C66",
```

Comment:
This is fine. I don't know how you're generating these, but you should actually try to change these to real hashed and salted passwords in the future. You can generate them using whatever function you use when you write the signup flow, and then you can test them using the login flow. Would be best to not have these be arbitrary, that way we can test both "user creation" and "user authentication" on the same data. Know what I mean?